### ISSUE TITLE
source does source from current working directory

### ISSUE THREAD

**Issue by 875d** [ahrm/sioyek#504]
If you have `source` in your `prefs_user.config`, the file is source from the current working directory instead of being sourced from `~/.config/sioyek`.

Consider the following example:

```
~/Documents 
❯ pwd
/home/USERNAME/Documents

❯ cat ~/.config/sioyek/prefs_user.config
[...]
source another_file.config

~/Documents 
❯ sioyek article.pdf 
[...]
**Error: Could not open config file /home/USERNAME/Documents/another_file.config**
[...]
```

**Comment by ahrm** [comment]
You need to use absolute positions.

**Comment by ahrm** [comment]
Should be fixed in ff5bcd30b18508fc5def03c3f2aff87f3c8c4282.

**Comment by 875d** [comment]
Thanks.

Doesn't QT have a function that expands file paths? Like python Path.absolute()?

**Comment by ahrm** [comment]
I don't know about Qt but c++ already has a pretty good `filesystem` module. Unfortunately we can't use that because there is a bug in the `gcc` included in `Ubuntu 18.04` (which is the platform we use to build linux builds) which makes it crash sometimes.

**Comment by 875d** [comment]
oh  I'm against supporting anything which is older than 1 year.

Thanks!
